test/games/dread/gui/test_dread_game_export_dialog.py

```diff
@@ -113,6 +113,7 @@ def test_on_input_file_button(skip_qtbot, tmp_path, mocker):
     tmp_path.joinpath("existing-folder").mkdir()
 
     for p in [
+        (".", "config.ini"),
```

Comment:
Could've done `("config.ini",)` to force it be a tuple.

Reply:
I swear i tried that earlier and it didn't work

Reply:
Oh, I missed the comma...